release-notes/EPM/I-DOCSIS/EPM_7.0.3.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: EPM_7.0.3_I-DOCSIS
+---
+
+# EPM 7.0.3 I-DOCSIS (preview)
+
+> [!IMPORTANT]
+> We are still working on this release. Release notes may still be added, modified, or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### EPM_I_DOCSIS_AddNewCcapCmPair improvements [ID_39450]
+
+Several improvements have been implemented in the EPM_I_DOCSIS_AddNewCcapCmPair script, which is used to create a new CCAP.
+
+- Every page now includes a *Cancel* button, which you can use the terminate the script.
+- Validation on fields now prevents the user from proceeding if fields still need to be filled in.
```

Comment:
Hi,
Maybe we could rephrase this sentence? If I understand correctly the script now validates empty fields. If this is the case we can just say *The script now validates empty fields before to proceed with the provisioning*

Reply:
How about this?
```suggestion
- If fields that must be filled in are still empty, the field validation will now prevent the user from proceeding.
```

Reply:
It looks ok. Thanks @MariekeGO!